usecases/scheduler/src/infra/service_screener_automation/service_screener_automation_stack.py

```diff
@@ -0,0 +1,271 @@
+from aws_cdk import (
+    Duration,
+    Stack,
+    Size,
+    RemovalPolicy,
+    aws_ec2 as ec2,
+    aws_batch as batch,
+    aws_ecs as ecs,
+    aws_events as events,
+    aws_events_targets as targets,
+    aws_iam as iam,
+    aws_s3 as s3,
+    aws_dynamodb as dynamodb,
+    aws_lambda as lambda_,
+    aws_lambda_event_sources as eventsources,
+    aws_scheduler as scheduler,
+    CfnOutput
+)
+from aws_solutions_constructs.aws_eventbridge_lambda import EventbridgeToLambda
+from aws_cdk.aws_lambda_python_alpha import (
+    PythonFunction,
+)
+from os import path, environ
+from constructs import Construct
+
+class ServiceScreenerAutomationStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        dirname = path.dirname(__file__)
+        super().__init__(scope, construct_id, **kwargs)
+        bucket_name = environ["BUCKET_NAME"]
+        prefix="Screener"
+
+        vpc = ec2.Vpc(self, "VPC",
+                    max_azs=2,
+                    nat_gateways=0,
+                    subnet_configuration=[
+                        ec2.SubnetConfiguration(
+                            name="Public",
+                            subnet_type=ec2.SubnetType.PUBLIC,
+                            cidr_mask=24
+                        )
+                        # ),
+                        # ec2.SubnetConfiguration(
+                        #     name="Isolated",
+                        #     subnet_type=ec2.SubnetType.PRIVATE_ISOLATED,
+                        #     cidr_mask=24
+                        # )
+                    ]
+        
+        )
+        #add check for existing bucket
+        #S3 Bucket
+        bucket = s3.Bucket(self, bucket_name,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Bucket is not set to enforce encryption-in-transit, if not explictly setting this on the bucket policy - the property &#34;enforce_ssl&#34; should be set to True

Suggested remediation:
Checks if the property `enforce_ssl` is set to True 
```
@@ -57,6 +57,5 @@
             versioned=True,
             removal_policy=RemovalPolicy.RETAIN,
-            event_bridge_enabled=True
-        )
+            event_bridge_enabled=True, enforce_ssl=True)
         #Create Batch
         batch_compute_env = batch.FargateComputeEnvironment(self, &#34;ScreenerEnv&#34;,

```